api/src/main/java/org/openmrs/module/pihcore/PihCoreActivator.java

```diff
@@ -95,8 +95,8 @@ public void started() {
                 config = Context.getRegisteredComponents(Config.class).get(0); // currently only one of these
             }
 
+			LiquibaseSetup.setup();
```

Comment:
I moved this up to happen before the Iniz setup, as I'll need it to clear out the location tag maps before the Iniz setup runs.

(Tangent: it occurs to me that it would be easy and helpful to add support for liquibase changesets that run both before and after metadata is loading... however, note that even before my change the liquibase changesets run before concept loading... )

Reply:
Sounds fine, but I just assumed you would add columns for every location tag to the csv, with null/empty values, and it would take care of doing all of the necessary cleanup since all columns would be present.

Reply:
@mseaton  yeah, that was my original idea, but then I realized that wouldn't work, because if a new location was added (or any change was made) to locations.csv, on the next restart all the tags would be wiped again when locations.csv was re-processed... and unless changes were also made to the locationtagmaps, the tag files would not get reprocessed.

Reply:
@mogoodrich I wonder if we want to configure the locationtagmaps domain to run "always", and ignore checksum.  Would that solve the issue?

Reply:
If that's a possible configuration, then, yeah, that would work... I think actually we'd want "always" for most metadata (except those that are time-consuming to load)...

Reply:
Looking at the docs, it appears you can specify a runtime property that indicates to skip checksums globally (and reload everything), but not on a domain-by-domain basis...